test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go

```diff
@@ -28,6 +28,7 @@ func TestNamedVectors_SingleNode(t *testing.T) {
 	}()
 	endpoint := compose.GetWeaviate().URI()
 	t.Run("tests", allTests(t, endpoint))
+	t.Run("legacy tests", allLegacyTests(endpoint))
```

Comment:
can we run `allLegacyTests` as part of `allTests` so that those tests will be run  also in single node / multi node / single node async / multi node async configurations?

Reply:
added `allLegacyTests` calls to remaining test suites